README.adoc

```diff
@@ -1158,6 +1164,17 @@ a $A $B=`echo $A`:
   echo $A $B
 ```
 
+Note: the setting link:README.adoc#export[export] causes all `just` variables to be exported to recipes as enviroment variables.
+
+===== Using an `.env` file
+
+`just` can load an .env file if the setting link:README.adoc#dotenv-load[dotenv-load] is present. The variables in the file will be available as environment variables to the recipes. See link:README.adoc#dotenv-integration[here] for more information.
+
+==== Loading `just` variables from the shell environments variables
+
+Environment variables from the shell can be loaded as variables in `just` with the functions `env_var()` and `en_var_or_default()`, described link:README.adoc#environment-variables[here].
+They could then be exported to the recipes using link:README.adoc#exporting-just-variables[export].
```

Comment:
I think that second part is redundant, because if they're loaded using `env_var`, they will already be exported to recipes by default.

```suggestion
==== Setting `just` Variables from Environments Variables

Environment variables can propagated to `just` variables using the functions `env_var()` and `env_var_or_default()`. See link:README.adoc#environment-variables[environment-variables].
```

Reply:
edited